cassandra-plugins/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>io.cdap.plugin</groupId>
     <artifactId>hydrator-plugins</artifactId>
-    <version>2.9.2</version>
+    <version>2.9.4-SNAPSHOT</version>
```

Comment:
The only version that should have changed was for Database plugins. No other artifact was released.